libs/shared/interfaces/src/lib/domain/job-application.entity.ts

```diff
@@ -0,0 +1,40 @@
+import {
+  ApplicationAdvancedSettings,
+  Commit,
+  DeploymentHistoryApplication,
+  Instance,
+  JobResponse,
+  Link,
+  Status,
+} from 'qovery-typescript-axios'
+import { LoadingStatus } from '../types/loading-status.type'
+import { ServiceRunningStatus } from './service-running-status.interface'
+
+export interface JobApplicationEntity extends JobResponse {
```

Comment:
Why not JobEntity ?

Reply:
Because it's the same pattern with other application

`job-application.entity.ts` 
`git-application.entity.ts`
`container-application.entity.ts`